docs/user/troubleshooting/03-70-reconciliation-fails-on-istio-install.md

```diff
@@ -0,0 +1,19 @@
+# Istio ingress-gateway Deployment, istiod Deployment or Istio CNI DaemonSets are not reconciling back to the desired state after being modified
+
+## Symptoms
+
+- The Istio Ingress Gateway Deployment, Istiod Deployment, or istio-cni-node DaemonSets are not getting up
+- A non-default container is present in any of the above, and cannot start
+- Istio Custom Resource is in Error state
+
+## Causes
+
+- The Istio Ingress Gateway Deployment, Istiod Deployment, or istio-cni-node DaemonSets might have a container injected by an outside component, for example with a mutating webhook. If the new container cannot start for any reason, the pod will not get to a Running state, failing Istio module reconciliation.
+
+## Remedy
+
+1. Check if a given resource's pod template is modified with additional container
```

Comment:
```suggestion
1. Check if a given resource's Pod template has been modified to include an additional container.
```

Reply:
Don't you want to add commands with placeholders to these steps so the user can just run them? If possible, of course. Now, we tell what to do but not how to do it